modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -35,26 +36,32 @@
     @Parameterized.Parameters(name = "{index} - {0}")
     public static Iterable<Object[]> data() {
         return asList(
-                new Object[][]{
-                        {"jdbc:tc:mysql:5.5.43://hostname/databasename", false, false, false},
-                        {"jdbc:tc:mysql://hostname/databasename?TC_INITSCRIPT=somepath/init_mysql.sql", true, false, false},
-                        {"jdbc:tc:mysql://hostname/databasename?TC_INITFUNCTION=org.testcontainers.jdbc.JDBCDriverTest::sampleInitFunction", true, false, false},
-                        {"jdbc:tc:mysql://hostname/databasename?useUnicode=yes&characterEncoding=utf8", false, true, false},
-                        {"jdbc:tc:mysql://hostname/databasename", false, false, false},
-                        {"jdbc:tc:mysql://hostname/databasename?useSSL=false", false, false, false},
-                        {"jdbc:tc:postgresql://hostname/databasename", false, false, false},
-                        {"jdbc:tc:mysql:5.6://hostname/databasename?TC_MY_CNF=somepath/mysql_conf_override", false, false, true},
-                });
+            new Object[][]{
+                {"jdbc:tc:mysql:5.5.43://hostname/databasename", false, false, false},
+                {"jdbc:tc:mysql://hostname/databasename?TC_INITSCRIPT=somepath/init_mysql.sql", true, false, false},
+                {"jdbc:tc:mysql://hostname/databasename?TC_INITFUNCTION=org.testcontainers.jdbc.JDBCDriverTest::sampleInitFunction", true, false, false},
+                {"jdbc:tc:mysql://hostname/databasename?useUnicode=yes&characterEncoding=utf8", false, true, false},
+                {"jdbc:tc:mysql://hostname/databasename", false, false, false},
+                {"jdbc:tc:mysql://hostname/databasename?useSSL=false", false, false, false},
+                {"jdbc:tc:postgresql://hostname/databasename", false, false, false},
+                {"jdbc:tc:mysql:5.6://hostname/databasename?TC_MY_CNF=somepath/mysql_conf_override", false, false, true},
+                {"jdbc:tc:mariadb:10.1.16://hostname/databasename", false, false, false},
+                {"jdbc:tc:mariadb://hostname/databasename", false, false, false},
+                {"jdbc:tc:mariadb://hostname/databasename?useUnicode=yes&characterEncoding=utf8", false, true, false},
+                {"jdbc:tc:mariadb://hostname/databasename?TC_INITSCRIPT=somepath/init_mariadb.sql", true, false, false},
+                {"jdbc:tc:mariadb://hostname/databasename?TC_INITFUNCTION=org.testcontainers.jdbc.JDBCDriverTest::sampleInitFunction", true, false, false},
+                {"jdbc:tc:mariadb:10.1.16://hostname/databasename?TC_MY_CNF=somepath/mariadb_conf_override", false, false, true}
```

Comment:
Haven't added Oracle here due to different SQL syntax needed for validation (further consolidation/refactoring could follow).